<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondarySkillProgressBar</class>
 <widget class="QWidget" name="SecondarySkillProgressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="Name">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NON</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ProgressBar_1" name="progressBar" native="true">
     <property name="minimumSize">
      <size>
       <width>734</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>734</width>
       <height>42</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>474</width>
       <height>13</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>474</width>
       <height>13</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="TipLeft">
        <property name="minimumSize">
         <size>
          <width>19</width>
          <height>11</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>19</width>
          <height>11</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="Border">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="TipRight">
        <property name="minimumSize">
         <size>
          <width>19</width>
          <height>11</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>19</width>
          <height>11</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressBar_1</class>
   <extends>QWidget</extends>
   <header location="global">CustomWidgets/ProgressBar_1/progressbar_1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
